aptos-move/aptos-transactional-test-harness/tests/aptos_test_harness/call_function.move

```diff
@@ -0,0 +1,35 @@
+//# init --addresses Alice=0xf75daa73fc071f93593335eb9033da804777eb94491650dd3f095ce6f778acb6
+//#      --private-keys Alice=56a26140eb233750cd14fb168c3eb4bd0782b099cde626ec8aff7f3cceb6364f
+
+//# publish
+module Alice::HelloWorld {
+    use AptosFramework::Signer;
+    use AptosFramework::Coin;
+    use AptosFramework::TestCoin::TestCoin;
+    use AptosFramework::ASCII::{Self, String};
+
+    struct ModuleData has key, store {
+        global_counter: u64,
+        state: String,
+    }
+
+    fun init_module(sender: &signer) {
+        move_to(
+            sender,
+            ModuleData { global_counter: 0, state: ASCII::string(b"init") }
+        );
+    }
+
+    public fun foo(addr: address): u64 {
+        Coin::balance<TestCoin>(addr)
+    }
+
+    public(script) fun hi(sender: &signer, msg: String) acquires ModuleData {
+        borrow_global_mut<ModuleData>(Signer::address_of(sender)).state = msg;
+    }
+}
+
+
+//# run --signers Alice --args x"68656C6C6F20776F726C64" --show-events -- Alice::HelloWorld::hi
+
+//# view --address Alice --resource Alice::HelloWorld::ModuleData
```

Comment:
this is very cool. 
just want to know your thought on supporting writing assert directly in the move file with ==, >=, <=, > and < operators. 
For example, 
//# view --address Alice --resource Alice::HelloWorld::ModuleData  --field_name="global_counter" --comparator=larger_than --target_value=0

Reply:
Can you elaborate more on how to do it through the Move script? But we can get that working in a separate PR.

Reply:
I am also curious how to do this assertion in move. If I have a move script after `view --address Alice --resource Alice::HelloWorld::ModuleData`. This move script won't be able to access the struct fields defined other module. We probably can only parse the resource